src/settings.h

```diff
@@ -123,6 +123,11 @@ class Settings
          */
         bool    onlyShowIconOnUnreadMessages;
 
+        /**
+         * Whether to log to console (stderr) or use line buffer
+         */
+        bool    mLogToConsole;
+
```

Comment:
Is there any reason we need this as a setting? I only see it useful for debugging (either by devs or users), why a setting?